ansible/roles/ocp4-workload-rhtr-iot-managed/tasks/remove_workload.yml

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
I'm not especially worried about this since I don't expect it will be used much, but please change all instances in here from `k8s_facts` to `k8s_info`.